### PR TITLE
Only warn once for bad R6 fields, methods, bindings

### LINKED ISSUE #934: Have a single warning for undocumented R6 methods, fields, bindings...
The many warnings can be annoying.

### PR DESCRIPTION
**Pull request by gaborcsardi**
Closes #934.

### CODE REVIEW

File: R/rd-r6.R [hunk 1]
```diff
@@ -445,3 +415,16 @@ r6_all_examples <- function(block, methods) {
...
   }))
 }
+
+first_five <- function(x) {
+  x <- encodeString(x, quote = "`")
+  if (length(x) > 5) x <- c(x[1:5], "...")
+  paste(x, collapse = ", ")
+}
+
+r6_warning <- function(template, block, bad) {
```
Comment (hadley) [inline R/rd-r6.R]: Can you please make `block` the first argument so it's consistent with `roxy_tag_warning()`?

Comment (gaborcsardi) [reply R/rd-r6.R]: Sure.

Comment (gaborcsardi) [reply R/rd-r6.R]: Done.

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**